script/redirect-cc.py

```diff
@@ -22,6 +23,13 @@ def main():
     return cc_retcode
 
 
+def append_root_src_include(args):
+  for arg in args:
+    if arg.startswith('-I') and arg.endswith('brave/chromium_src'):
+        args.append(arg + '/../../..')
```

Comment:
this arg append allows us to put `-I<path>` at the end for compiler args which will not break any of the cases when overlaps are possible, for example: `base/macros.h` and `v8/src/base/macros.h`. Modifying `include_dirs` in the compiler config here won't work:
https://github.com/brave/brave-core/blob/7f68143a2cdca2af447e678e5a25ba15d832ef16/build/BUILD.gn#L4

Reply:
Also I tried to do this only for `brave/chromium_src` files, but this won't work, because non-overridden files still need this include path.